crates/sui-indexer-alt-framework/src/pipeline/concurrent/committer.rs

```diff
@@ -389,11 +412,12 @@ mod tests {
         };
 
         let batch2 = BatchedRows {
-            values: vec![StoredData {
+            batch: vec![StoredData {
                 cp_sequence_number: 3,
                 tx_sequence_numbers: vec![7, 8, 9],
                 ..Default::default()
             }],
+            batch_len: 2,
```

Comment:
Is this right? The `batch` vector has only one row, but the `batch_len` is 2. I think there are a couple of other cases of this. Maybe it's worth creating a helper function for creating a `BatchedRows` from a vector.